addons/l10n_ar/models/account_move.py

```diff
@@ -360,11 +361,62 @@ def _l10n_ar_get_invoice_totals_for_report(self):
             for subtotal in tax_totals['subtotals']
             for tax_group in subtotal['tax_groups']
         }
-        tax_group_ids_to_exclude = self.env['account.tax.group'].browse(tax_group_ids).filtered('l10n_ar_vat_afip_code').ids
+        tax_group_ids_to_exclude = self.env['account.tax.group']\
+            .browse(tax_group_ids)\
+            .filtered(lambda tax_group: (
+                not self._l10n_ar_is_tax_group_other_national_ind_tax(tax_group)
+                and not self._l10n_ar_is_tax_group_vat(tax_group)
```

Comment:
We have a problem, we are showing the ones that we need to hide (the vat taxes and national and indirect taxes)
Check Image for functional test
![image](https://github.com/user-attachments/assets/69d96b27-a95f-40be-840c-88c218a1ea45)


I think that maybe this change will solve it.

```suggestion
            .filtered(lambda tax_group: (
                self._l10n_ar_is_tax_group_other_national_ind_tax(tax_group)
                or self._l10n_ar_is_tax_group_vat(tax_group)
```

Reply:
@smetl Moreover, I found something else: We are not showing the "Other Tax" in the breakdown.
This "Other Tax" has a specific tax group that should not go into the new table, but displayed in the breakdown with the total:
<img width="976" alt="image" src="https://github.com/user-attachments/assets/dc29c263-3f03-4778-966d-c505ab8ba095" />
<img width="431" alt="image" src="https://github.com/user-attachments/assets/ad80e565-3deb-41cb-9076-2f3558776d5d" />
cc' @zaoral